Libraries/pbxbuild/Sources/Tool/CopyResolver.cpp

```diff
@@ -25,13 +26,15 @@ CopyResolver(pbxspec::PBX::Tool::shared_ptr const &tool) :
 {
 }
 
-void Tool::CopyResolver::
-resolve(
+static void
+ResolveInternal(
+    pbxspec::PBX::Tool::shared_ptr const &tool,
     Tool::Context *toolContext,
     pbxsetting::Environment const &baseEnvironment,
-    std::vector<Phase::File> const &inputs,
+    std::vector<std::string> const &inputPaths,
+    ext::optional<std::vector<Phase::File>> const &inputs,
     std::string const &outputDirectory,
```

Comment:
Completely random aside:  I've seen in Buck that strings that represent paths are passed around in their own object type called a SourcePath.  Is there anything similar in xcbuild?